nilearn/input_data/confounds_utils.py

```diff
@@ -0,0 +1,250 @@
+"""Helper functions for the manipulation of confounds.
```

Comment:
I think that the dependency of all this on fmriprep needs to be made explicit, even possibly in module/function name.
AFAIK not everybody uses fmriprep and this part of the codebase has a lot of implicit assumption on fmriprep outputs.

Reply:
@pbellec and I talked about this a bit. It should be more explicit from the module or function name. Instead of using `confounds` as prefix, I thought about `fmriprep` and adjust some of the module names accordingly. For the main function, maybe we should use `fmriprep_confounds` or `fmriprep_denoise` instead of `load_confounds`
WDYT?

Reply:
After sleeping on it for a while, I am absolutely in favor of making the tie with `fmriprep` explicit. Maybe in the future this code will support any pipeline compliant with `bids` derivatives, but in the meantime the code is specifically designed to operate on `fmriprep` outputs (even supporting multiple versions of this software). I would be in favor of using `fmriprep_confounds` rather than `fmriprep_denoise`, because `confounds` is the name used throughout the documentation of `fmriprep` and strictly speaking the confounds do not perform denoising by themselves, but are rather fed into a denoising procedure (`signal.clean`).

Reply:
I think we're on the same page (and indeed, I changed my mind over this question). `fmriprep_confounds` is probably the best term.